res/values-pl/strings.xml

```diff
@@ -120,5 +120,6 @@
     <string name="flip_action_dialog_title">Akcja odwrócenia</string>
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit